src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/X509Certificates/X509CertificateKeyAccessors.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace System.Security.Cryptography.X509Certificates
+{
+    /// <summary>
+    /// Helper methods to access keys on <see cref="X509Certificate2"/>.
+    /// </summary>
+    public static class X509CertificateKeyAccessors
```

Comment:
Maybe we should EB-Never this for .NET 10+?

Reply:
Would you move the EB-Never to .NET 11 if we add support for HQC next year; or is your thinking more "this exists for .NET Framework and .NET Standard, and anyone on the .NET line should just use the latest version"?

I'd be more behind `#if NET` than `#if NET_10_OR_GREATER`, just because it takes away the "when do we bump this?"

Reply:
Though making it EB-Never is probably an API Review thing to discuss :)

Reply:
That's fair. Perhaps EB-never on the individual extension methods then, and we just have a policy of EB-Ning them once they are available on .NET proper.